pkgs/applications/misc/crumbs/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, automake, autoconf, gzip, ruby, bash }:
+
+stdenv.mkDerivation rec {
+  name = "crumbs";
+  version = "0.0.2";
+  isExecutable = true;
+
+  src = fetchFromGitHub
+    { owner = "fasseg";
+      repo = "crumbs";
+      rev = "2ac7a909e66cff019acaa0566be721e3696f0bce";
+      sha256 = "1m7fwjw74fy8pgsvjda1c7awl2y5jr0r3fb5a6v5rddym5w52s0b";
+    }; 
+
+  configurePhase = ''
+    automake --add-missing
+    ./configure --disable-dependency-tracking --prefix=$out
+  '';
+
+  buildInputs =
+    [ automake
+      autoconf
```

Comment:
automake and autoconf are executed when building, not at runtime, so they should go in `nativeBuildInputs` instead of `buildInputs`.
In any case, if you replace them by `autoreconfHook`, you can get rid of the custom `configurePhase`.